9c-main/chart/templates/empty-chronicle.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: empty-chronicle
+  labels:
+    app: empty-chronicle
+    app.kubernetes.io/instance: {{ $.Chart.Name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: empty-chronicle
+  serviceName: empty-chronicle
```

Comment:
It may need to create `Service` resource too 👀 

Reference https://github.com/planetarium/9c-infra/blob/311fd2db01c1cdf6fe2e0e08c52c3cc602f2a23f/9c-main/chart/templates/service.yaml#L36-L73

Reply:
You can see https://kubernetes.io/ko/docs/concepts/services-networking/service/